data/secscan_model/secscan_v4_model.py

```diff
@@ -62,12 +81,13 @@ def __init__(self, app, instance_keys, storage):
 
         validator = V4SecurityConfigValidator(
```

Comment:
Can we add a check above these that sees if the SECURITY_SCANNER_V4_ENDPOINT is `None`, and if so, not write any warning log?